GVFS/GVFS.UnitTests.Windows/GVFS.UnitTests.Windows.csproj

```diff
@@ -181,8 +184,314 @@
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\GVFS.UnitTests\**\*.cs">
-      <Link>NetCore\%(RecursiveDir)\%(Filename)%(Extension)</Link>

```

Comment:
>Why did this recursive entry have to get broken out to all of the individual files?

VS goodness? :)  I will revert these changes.

Reply:
I think this might happen if you try to add a file that lives in `GVFS.UnitTests` into `GVFS.UnitTests.Windows.csproj ` directly rather than adding it to `GVFS.UnitTests` and then reloading the `GVFS.UnitTests.Windows` project.